File: JAVA/Aksesoris.java
class Aksesoris extends PetShop {
    private String jenis;
    private String bahan;
    private String warna;

    public Aksesoris() {
        super();
        this.jenis = "";
        this.bahan = "";
        this.warna = "";
    }

    public Aksesoris(int id, String namaProduk, int hargaProduk, int stokProduk, String jenis, String bahan, String warna) {
        super(id, namaProduk, hargaProduk, stokProduk);
        this.jenis = jenis;
        this.bahan = bahan;
        this.warna = warna;
    }

    public void setJenis(String jenis) { this.jenis = jenis; }
    public void setBahan(String bahan) { this.bahan = bahan; }
    public void setWarna(String warna) { this.warna = warna; }

    public String getJenis() { return this.jenis; }
    public String getBahan() { return this.bahan; }
    public String getWarna() { return this.warna; }
}

File: PYTHON/Aksesoris.py
from PetShop import PetShop
class Aksesoris(PetShop):
    def __init__(self, id=0, namaProduk="", hargaProduk=0, stokProduk=0, jenis="", bahan="", warna=""):
        super().__init__(id, namaProduk, hargaProduk, stokProduk)
        self.jenis = jenis
        self.bahan = bahan
        self.warna = warna
    
    def setJenis(self, jenis):
        self.jenis = jenis
    def setBahan(self, bahan):
        self.bahan = bahan
    def setWarna(self, warna):
        self.warna = warna
    
    def getJenis(self):
        return self.jenis
    def getBahan(self):
        return self.bahan
    def getWarna(self):
        return self.warna